server/utils/tasks/listTable.ts

```diff
@@ -18,18 +18,27 @@ const statusCell = (task: Task): TableCell => ({
   html: statusBadge(task),
 })
 
-const getTaskType = (taskType: string): string => {
-  if (taskType === 'PlacementRequest') {
-    return 'Match request'
+const getTaskType = (task: Task): string => {
+  let taskType
+  if (task.taskType === 'PlacementRequest') {
+    taskType = 'Match request'
   }
-  if (taskType === 'PlacementApplication') {
-    return 'Request for placement'
+  if (task.taskType === 'PlacementApplication') {
+    taskType = 'Request for placement'
   }
-  return sentenceCase(taskType)
+  if (task.taskType === 'Assessment') {
+    const assessmentTask = task as AssessmentTask
```

Comment:
This is fine but it could be a good candidate for a type assertion like `isAssessment

Reply:
Good call!

Reply:
I've added those now, with some tests. Makes the function a bit neater too.